Gruntfile.js

```diff
@@ -97,7 +97,9 @@ module.exports = function(grunt) {
             'effect': 'src/effect',
             'filter': 'src/filter',
             'reverb': 'src/reverb',
+            'eq': 'src/eq',
             'distortion': 'src/distortion',
+            'compressor': 'src/compressor',
```

Comment:
sorry about this! must have merged a branch by accident-- will commit this compresser stuff out of here